templates/partials/footer.html

```diff
@@ -2,31 +2,31 @@
     <div class="mzp-l-content">
         <nav class="mzp-c-footer-primary">
             <div class="mzp-c-footer-primary-logo">
-                <a href="https://mozilla.org">Mozilla</a>
+                <a href="https://www.mozilla.org">Mozilla</a>
             </div>
             <div class="mzp-c-footer-sections">
                 <section class="mzp-c-footer-section">
                     <h5 class="mzp-c-footer-heading">
                         Company
                     </h5>
                     <ul class="mzp-c-footer-list">
-                        <li><a href="https://mozilla.org/en-US/about/manifesto/">Mozilla Manifesto</a></li>
+                        <li><a href="https://www.mozilla.org/about/manifesto/">Mozilla Manifesto</a></li>
                         <li><a href="https://blog.mozilla.org/press/">Press Center</a></li>
                         <li><a href="https://blog.mozilla.org/">Corporate Blog</a></li>
 
-                        <li><a href="/en-US/careers/" data-link-type="footer" data-link-name="Careers">Careers</a></li>
-                        <li><a href="/en-US/contact/" data-link-type="footer" data-link-name="Contact">Contact</a></li>
+                        <li><a href="https://www.mozilla.org/careers/" data-link-type="footer" data-link-name="Careers">Careers</a></li>
+                        <li><a href="https://www.mozilla.org/contact/" data-link-type="footer" data-link-name="Contact">Contact</a></li>
                         <li><a href="https://foundation.mozilla.org/?form=donate">Donate</a></li>
                     </ul>
                 </section>
                 <section class="mzp-c-footer-section">
                     <h5 class="mzp-c-footer-heading">Resources</h5>
                     <ul class="mzp-c-footer-list">
-                        <li><a href="https://mozilla.org/en-US/privacy/">Privacy Hub</a></li>
+                        <li><a href="https://www.mozilla.org/privacy/">Privacy Hub</a></li>
 
-                        <li><a href="https://mozilla.org/en-US/firefox/browsers/compare/">Browser Comparison</a></li>
+                        <li><a href="https://www.mozilla.org/firefox/browsers/compare/">Browser Comparison</a></li>
 
-                        <li><a href="https://mozilla.design/">Brand Standards</a></li>
+                        <li><a href="https://www.mozilla.design/">Brand Standards</a></li>
```

Comment:
this one doesn't have any ws
```suggestion
                        <li><a href="https://mozilla.design/">Brand Standards</a></li>
```